addons/calendar/models/calendar_event.py

```diff
@@ -762,7 +902,7 @@ def split_privacy(events):
             public = [event for event in events if event.get('privacy') != 'private']
             return private, public
 
-        def my_events(events):
+        def split_visibility(events):
```

Comment:
Warning, the method my_events should remains even if it just call my_events. You could break other module or customization.